runtime/turing/src/lib.rs

```diff
@@ -288,6 +288,23 @@ impl frame_system::Config for Runtime {
 	type MaxConsumers = ConstU32<16>;
 }
 
+parameter_types! {
+	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
```

Comment:
Trying to understand where these numbers are from. My reading of the docs come out differently.
```
/// This is held for an additional storage item whose value size is
/// `4 + sizeof((BlockNumber, Balance, AccountId))` bytes and whose key size is
/// `32 + sizeof(AccountId)` bytes.
```
https://github.com/paritytech/substrate/blob/4a19d408095ebcb47840ad1f2f086f53a0020cb7/frame/multisig/src/lib.rs#L134-L136

4 + sizeof((BlockNumber, Balance, AccountId)) = 4 + sizeof((u32, u128, u32(?))) = 4 + 4 + 16 + 4 = 28 bytes
32 + sizeof(AccountId) = 32 + 4 = 36 bytes
total = 28 + 36 = 64 bytes

Reply:
OK, I'll go look up some sources and understand it.
If I figure it out, I'll reply.